uitools/examples/cpp_quick/src/proxies/EnumsProxy.h

```diff
@@ -0,0 +1,26 @@
+#ifndef ARCGIS_RUNTIME_TOOLKIT_CPP_QUICK_DEMO_ENUMSPROXY_H
+#define ARCGIS_RUNTIME_TOOLKIT_CPP_QUICK_DEMO_ENUMSPROXY_H
+
+#include <QObject>
+
+#include <MapTypes.h>
+
+class EnumsProxy : public QObject
+{
+  Q_OBJECT
+public:
+  Q_INVOKABLE explicit EnumsProxy(QObject* parent = nullptr);
+  ~EnumsProxy() override;
+
+  enum class LoadStatus 
+  {
+    LoadStatusLoaded = static_cast<int>(Esri::ArcGISRuntime::LoadStatus::Loaded),
```

Comment:
we should just get these exposed to qml from the api if users will need them

Reply:
I'd love to see this proxies folder removed in its entireity. Currently, I think `ArcGISRuntimeEnvironment.apiKey` is the lowest hanging fruit. Enumerations might require a bit of consideration in how to expose to QML because of how we define them in C++.